src/dsp_tools/utils/connection_live.py

```diff
@@ -298,6 +304,9 @@ def _handle_non_ok_responses(self, response: Response, request_url: str, retry_c
             msg = "Permanently unable to execute the network action. "
             if original_str := regex.search(r'{"knora-api:error":"dsp\.errors\.(.*)","@context', str(response.content)):
                 msg += f"\n{' '*37}Original Message: {original_str.group(1)}\n"
+                msg += f"See logs for more details: {logger_savepath}"
```

Comment:
In the case where the following if-statement is false, this line is executed again on line 310 or am I mistaken here? And if so is this a desired behavior?

Reply:
Good catch, thank you. You're right, that's not intended.